frontend/src/component/common/PageContent/PageContent.styles.ts

```diff
@@ -9,9 +9,7 @@ export const useStyles = makeStyles()(theme => ({
         [theme.breakpoints.down('md')]: {
             padding: theme.spacing(2),
         },
-        [theme.breakpoints.down('sm')]: {
-            overflowX: 'auto',
-        },
+        overflowX: 'auto',
```

Comment:
Seems like `PageContent` hasn't been migrated to StyledComponents yet. Having `overflowX: 'auto'` applied every time instead of only on the breakpoint doesn't seem to hurt and fixes the issue.